### ISSUE TITLE
Calculation of Gross Heating Value

### ISSUE THREAD

**Issue by ldovalxx** [usnistgov/REFPROP-issues#238]

**Comment by ldovalxx** [comment]
Good morning,

I can reformulate my question below. How is Refprop calculating Gross Heat Value (gas phase) in (BTU/ft3) for the following case (see picture below)? Which is the pressure and consequently the density is being used to calculate the volumetric heat value?

Thank you,

**Comment by ldovalxx** [comment]
Hi Mr. Lemmon,

Recently, my company have bought to me the last version 10.0 of Reprop. 

I have started to reproduce the gross heat value in btu/lbm, btu/ft3 and btu/gal of some components of natural gas liquids using Refprop with GPA 2145-2016 and I have found some differences that I would like to have your help if you can.

For example, I have attached a file showing my calculation with Refprop. Its running with pure Hexane, 60 F and 14.696 psia. The GHV in Btu/lbm is around the number from GPA (20943 btu/lbm). So, if I multiply 20958 BTU/lbm x 5.5362 lbm/gal (density) we can get 116,028 BTU/gal. This number is similar to GPA (115950 Btu/gal). But version 10.0 is showing me 636.22 BTU/gal. 

Also, for the second picture, if we multiply 20958 Btu/lbm x 41.414 lbm/ft3 (density) we can get 867,955 Btu/ft3. But, the GHV presented by version 10.0 is 4759.2 Btu/ft3 that is similar from the value of GPA 4755.9 Btu/ft3.

So, I would like to know if I am making some mistakes or wrong interpretation. Could you, please, clarify to me.

Thank you,
Luiz

**Comment by ldovalxx** [comment]
Note: Hexane @ 60 oF and 14.696 psia is in liquid phase.

**Comment by ldovalxx** [comment]
As per the above temp and pressure, Refprop has calculated the density as being 41.414 lbm/ft3 the Gross Heat Value of 20958 BTU/lbm and Gross heat value (volume basis) 4759.2 btu/ft3.

**Comment by EricLemmon** [comment]
Before answering your question, may I give a personal comment?  U.S. customary units (often now incorrectly called English or British units) are horrible!!!  I am the one writing the standards on fluid properties for GPA and AGA, including GPA2145-16 and its revision coming in a few years, and I think the folks in these meetings are very tired of hearing this same comment.  But I have sworn that before I die I will convince at least a small portion of them to transfer over to SI.  They tell me that I will have to live forever to see that happen.  So be it!

Having said that, trying to understand the conversions in 2145 is much easier if you start with the SI values.  I've pasted in a picture from Refprop for hexane at 15 C and .101325 MPa.  Also below is a snapshot of the values from GPA 2145.  In Refprop under the Options/Properties menu and then under the Special tab is an option for the ideal gas density.  That is also included in the table below.  Aside from the last digit, the numbers between Refprop and 2145 are nearly the same.  If you multiple the heating values by the ideal gas density, you will then get the last two items in the GPA table not shown in the Refprop screen shot.

More to come on English units in a second post...

![image](https://user-images.githubusercontent.com/42392984/72840979-a74c4600-3c52-11ea-96ac-c20e141ebeb3.png)
![image](https://user-images.githubusercontent.com/42392984/72841016-b92de900-3c52-11ea-974c-34eed0acdc05.png)

**Comment by ldovalxx** [comment]
Thank you for your feedback Mr. EricLemmon.
Another question,

Hexane at 15 oC and 101.325 kPa is in liquid phase. So, does it make sense to report the heat value as ideal gas?   What do you think?
Thank you,
Regards,
Luiz

**Comment by EricLemmon** [comment]
I've just posted a lengthy discussion concerning all of this at:

https://github.com/usnistgov/REFPROP-issues/issues/259

I hope this helps!  If you have questions about it, post them here.

**Comment by ldovalxx** [comment]
Thank you Dr. Eric!

Very clear and valuable technical inputs shown in the item# 259. It is an excellent technical reference.

Kind Regards,

Luiz

**Comment by ldovalxx** [comment]
Hi Dr. Eric,
I have an application to determine liquid Ethane density with good accuracy using an EOS.
The composition of the ethane stream is normally, for example:
Mix* = 99.235 Ethane, 0.51 Methane, 0.255 Propane or sometimes Mix* = 99.429 Ethane and 0.571 Methane. But the ethane purity is always > 99%. The pressure is ranging from 750 - 850 psig and temp: 45 - 65 oF. 
So, my question is: what is the best EOS (or approach) to be used to calculate the density of liquid ethane using Refprop? The EOS for pure Ethane assuming 100% of Ethane or EOS GERG 2008 assuming the mixtures described above for binary and ternary mixtures? I have made a comparison between these two EOS using Refprop and I have got a maximum difference between them of order of 1% for the above cases and conditions. Gerg TM 15 2007, is claiming an accuracy for ternary mixture of 0.1 to 0.5 % and 0.1 to 0.2% for the binary mixtures of C1 and C2. For pure Ethane we can get 0.03 to 0.05% for the above conditions for temp and pressure. 
Also, TM 15 test the binary and ternary mixtures with ethane concentration less than 99% (mole fraction).  
Thank you
Kind Regards,
Luiz
